Description: Don't strip symbols in upstream's Makefile, let dh_strip take care
 of stripping, which allows for debug builds.
Origin: https://bugs.launchpad.net/bugs/546108
Author: TJ <ubuntu@tjworld.net>
Last-Update: 2010-03-24
Forwarded: not-needed
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 all: dmg2img vfdecrypt 
 
 dmg2img: dmg2img.c dmg2img.h mntcmd.h gpt.h dmg2img.o base64.o adc.o 
-	$(CC) -s -o dmg2img dmg2img.o base64.o adc.o -L. -lz -lbz2
+	$(CC) -o dmg2img dmg2img.o base64.o adc.o -L. -lz -lbz2
 
 dmg2img.o: dmg2img.c dmg2img.h
 	$(CC) $(CFLAGS) -c dmg2img.c
@@ -17,11 +17,11 @@
 	$(CC) $(CFLAGS) -c adc.c
 
 vfdecrypt: vfdecrypt.c
-	$(CC) $(CFLAGS) -s -o vfdecrypt vfdecrypt.c -lcrypto
+	$(CC) $(CFLAGS) -o vfdecrypt vfdecrypt.c -lcrypto
 
 install: dmg2img vfdecrypt
 	mkdir -p ${BIN_DIR}
-	install -c -s -m 755 -o root -g root dmg2img vfdecrypt $(BIN_DIR)
+	install -c -m 755 -o root -g root dmg2img vfdecrypt $(BIN_DIR)
 
 clean:
 	rm -f dmg2img vfdecrypt *~ *.o core
